src/pages/team.en.tsx

```diff
@@ -1,188 +1,120 @@
+// https://ourcodeworld.com/articles/read/278/how-to-split-an-array-into-chunks-of-the-same-size-easily-in-javascript
+const chunkArray = (myArray: Array<any>, chunk_size: number) => {
+  var results = [];
 
-export const TeamPageScaffolding = (props: ContentfulContent) => (
-  <Layout metadata={props.content.metadata}>
-    <div id="team" className="team-page">
-      <section className="hero is-small">
-        <div className="hero-body has-text-centered is-horizontal-center">
-          <div className="container">
-            <h1 className="title is-size-2 has-text-grey-dark has-text-weight-normal is-spaced">
-              {props.content.title}
-            </h1>
-          </div>
-        </div>
-      </section>
+  while (myArray.length) {
+    results.push(myArray.splice(0, chunk_size));
+  }
 
-      <section className="content-wrapper team-members">
-        <div className="columns is-multiline">
-          {props.content.teamMembers.map((member: any, i: number) => (
-            <div
-              className="column team-member is-one-third has-text-centered "
-              key={i}
-            >
-              <figure className="image is-128x128 is-horizontal-center">
-                <img
-                  className="is-rounded"
-                  src={member.photo.fluid.src}
-                  alt={member.name}
-                />
-              </figure>
+  return results;
+};
 
-              <h4 className="title is-size-5 has-text-grey-dark">
-                {member.name}
-              </h4>
-              <h4 className="subtitle is-size-5 has-text-grey-dark">
-                {member.title}
-              </h4>
-              <div className="field">
-                {member.professionalLinks &&
-                  member.professionalLinks.map((link: any, i: number) => (
-                    <SocialIcon
-                      key={i}
-                      url={link}
-                      target="_blank"
-                      rel="noopener noreferrer"
-                      bgColor="#0096D7"
-                      style={{ height: 40, width: 40 }}
-                    />
-                  ))}
-              </div>
-              <p>{member.description.description}</p>
-            </div>
-          ))}
-        </div>
-      </section>
+type MemberCardInfo = {
+  name: string;
+  title: string;
+  description: {
+    description: string;
+  };
+  photo: {
+    fluid: {
+      src: string;
+    };
+  };
+};
 
-      <section className="hero is-small">
-        <div className="hero-body has-text-centered is-horizontal-center">
-          <div className="container">
-            <h1 className="title is-size-2 has-text-grey-dark has-text-weight-normal is-spaced">
-              {props.content.designAdvisorsTitle}
-            </h1>
-          </div>
-        </div>
-      </section>
+const MemberCard: React.FC<MemberCardInfo> = (props) => (
+  <div className="column is-3 is-paddingless">
```

Comment:
I would remove the `is-paddingless` class hear as I think it's creating some misalignment with the Figma designs. 
![image](https://user-images.githubusercontent.com/12834575/178268755-45bb83cf-4a75-4dee-a5be-7b14cbdb9e58.png)